requirements.txt

```diff
@@ -3,25 +3,11 @@
 -e .
 
 boto==2.39.0
-cffi==1.11.2
-cliff==2.2.0
-cryptography==2.2.1
+cryptography==2.7
 fasteners==0.14.1
-keystoneauth1==2.12.1
 idna==2.6
-openstacksdk==0.9.5
-os-client-config==1.21.1
-osc-lib==1.1.0
-oslo.config==3.17.0
-oslo.i18n==3.9.0
-oslo.serialization==2.13.0
-oslo.utils==3.16.0
 pycparser==2.18
-pyopenssl==16.2.0
-python-cinderclient==1.9.0
-python-glanceclient==2.5.0
-python-keystoneclient==3.5.0
-python-neutronclient==4.1.1
-python-novaclient==6.0.0
```

Comment:
Do we know which functions depend on these? (cinder, glance, keystone, and neutron)

Reply:
No, I have no idea, all those dependencies will be included by resolving python-openstackclient, later we can updated requirements with `pip-compile`, but this requires more activities and enable `pip-sync` in the `bootstrap` back.
But I was successfully run SES6 suse suite deployed from this branch using this requirements.txt